### PR TITLE
Installation and Start on a user specified port

### PR DESCRIPTION
**Pull request by SKrPl**
Solves issue: https://github.com/loklak/loklak_server/issues/925

### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by SKrPl** [comment]
@djmgit @vibhcool @singhpratyush @kavithaenair please review this :) .

**Comment by kavithaenair** [comment]
Does this work for windows well?

File: src/org/loklak/LoklakInstallation.java [hunk 1]
```diff
@@ -130,7 +130,10 @@ public static void main(String[] args) throws Exception {
         if(env.containsKey("PORTSSL")) {
             httpsPort = Integer.parseInt(env.get("PORTSSL"));
         }
-        
+
+        if (args.length > 0) {
+            httpPort = Integer.valueOf(args[0]);
```
Comment (vibhcool) [inline src/org/loklak/LoklakInstallation.java]: @djmgit see this -> http://stackoverflow.com/questions/7355024/integer-valueof-vs-integer-parseint

**Comment by djmgit** [comment]
@SKrPl config.properties has a property shortlink.urlstub=http://localhost:9000 so $LOCALHOST$CUSTOMPORT is just appending custom port at the end of http://localhost:9000

**Comment by SKrPl** [comment]
@kavithaenair no it doesn't work for windows. I will learn batch scripting and  do the necessary changes in `start.bat`, I couldn't find any `installation.bat` for installation in windows (How is loklak server installed in windows?).

@singhpratyush agree that way the startup time will also reduce.

@djmgit I will resolve that :) .

**Comment by mariobehling** [comment]
Please resolve, so it can be merged. Thanks.

**Comment by vibhcool** [comment]
@mariobehling, It is outputing error, this needs to be resolved

**Comment by singhpratyush** [comment]
Was this change even needed? Couldn't we just set the fields in `config.properties`?

**Pull request merged**